SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -7,6 +7,7 @@
 #include "Engine/BlueprintGeneratedClass.h"
 #include "Engine/SCS_Node.h"
 #include "GameFramework/Actor.h"
+#include "Kismet/KismetSystemLibrary.h"
```

Comment:
This still should be surrounded with
`#if WITH_EDITOR`
`#endif`

Reply:
Thanks! Done.